_posts/2025-04-19-navigating-pagination-with-hybrid-query.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: post
+title:  "Navigating pagination in hybrid queries with the pagination_depth parameter"
+authors:
+- varunudr
+- minalsha
+- seanzheng
+- kolchfa
+date: 2025-04-16
+categories:
+  - technical-posts
+meta_keywords: hybrid queries, semantic search, lexical search, search relevance, query execution workflow, OpenSearch hybrid search, Pagination
+meta_description: Explore how pagination works in hybrid queries by using pagination depth to efficiently paginate over the result set.
```

Comment:
Please update the meta with the following: 

_meta_keywords:_ hybrid query, OpenSearch, understanding pagination, pagination depth, hybrid search, search performance, best practices for using pagination_depth, query optimization, search relevance

_meta_description:_ Discover how to optimize OpenSearch hybrid queries using the pagination_depth parameter. Learn implementation guidelines and performance best practices for efficient search result navigation in OpenSearch 2.19.

Reply:
@pajuric Meta and link added.